packages/reporters/liquidation-reporter/excel-writer.ts

```diff
@@ -72,15 +72,18 @@ export function createExcelSheetFromLiquidationDrawDownData(drawDownData: any) {
         ? -1
         : 0
     );
+    console.log("a");
 
     collateralData.activeFinancialContracts.forEach((financialContractInfo: any, index: number) => {
-      sheet.cell(4 + index, 7).string(financialContractInfo.contractAddress);
-      sheet.cell(4 + index, 8).number(Number(Number(financialContractInfo.collateralValueInUsd).toFixed(2)));
-      sheet.cell(4 + index, 9).string(financialContractInfo.contractPriceIdentifier);
-      sheet.cell(4 + index, 10).date(new Date(financialContractInfo.contractExpirationTime * 1000));
-      sheet.cell(4 + index, 11).number(financialContractInfo.collateralRequirement);
+      sheet.cell(4 + index, 7).string(financialContractInfo.contractAddress || "unknown address");
```

Comment:
Is all of the `liquidation-reporter` work indepenednt of the multi-EOA stuff? Just want to make sure I shouldn't consider these as related changes

Reply:
o ye this should not have been committed. have reverted it. thanks for calling it out!